kadi/commands/command_sets.py

```diff
@@ -90,17 +91,27 @@ def cmd_set_scs107(date=None):
     cmds = cmd_set_end_observing()
     cmds += (
         dict(type="COMMAND_SW", dur=1.025, tlmsid="OORMPDS"),
-        dict(
-            type="COMMAND_HW",
-            # dur=1.025,
-            tlmsid="AFIDP",
-            msid="AFLCRSET",
-        ),
+        dict(type="COMMAND_HW", tlmsid="AFIDP", msid="AFLCRSET"),
         dict(type="SIMTRANS", params=dict(POS=-99616), dur=65.66),
         dict(type="ACISPKT", tlmsid="AA00000000", dur=1.025),
         dict(type="ACISPKT", tlmsid="AA00000000", dur=10.25),
         dict(type="ACISPKT", tlmsid=pow_cmd),
     )
+    # Potentially exclude HRC SCS-107 commanding for some regression testing. This is
```

Comment:
A nit and I know that comments are supposed to be there to help with code-gotchas, but here, leading with the regression testing sentence and starting with "potentially" makes it look like we're including the HRC commands for something to do with regression testing (even though the opposite is true).  I'd go with something more like:

Note that the following block to include HRC turn-off commands is wrapped in a config variable to make this conditional in support of regression testing.  This is needed for tests of command generation vs. commands in the archive which did not include this HRC SCS-107 commanding (prior to #344).

Reply:
Done.